src/script.cpp

```diff
@@ -1500,11 +1500,27 @@ bool is_stealth_info_type(const operation_stack& ops)
     return ops.size() == 2 &&
         ops[0].code == opcode::return_ &&
         ops[1].code == opcode::special &&
-        ops[1].data.size() >= hash_size;
+        ops[1].data.size() <= 40;
```

Comment:
From the [official developer guide](https://bitcoin.org/en/developer-guide#null-data):
> **Null Data (OP_RETURN) Transaction**
> Pubkey Script: OP_RETURN <0 to 40 bytes of data>

Also, some pools (e.g.[eligius](http://eligius.st/~gateway/faq-page)) allows up to 80 bytes of data:
> Data carrier transactions are permitted to be up to 80 bytes.